.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+      - name: Install Dependencies
+        run: npm ci
+      - name: Check Linting
+        run: npm run lint
+
+  tests:
+    runs-on: ubuntu-latest
+    name: Tests
+    steps:
+      - name: Fix usage of insecure GitHub protocol
+        run: sudo git config --system url."https://github".insteadOf "git://github"
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)